src/v2/client/sessionStorageHelper.js

```diff
@@ -12,14 +12,14 @@
 
 const STATE_HANDLE_SESSION_STORAGE_KEY = 'osw-oie-state-handle';
 
-const removeStateHandle = () => {
-  sessionStorage.removeItem(STATE_HANDLE_SESSION_STORAGE_KEY);
+const removeStateHandle = (appId) => {
+  sessionStorage.removeItem(`${STATE_HANDLE_SESSION_STORAGE_KEY}-${appId}`);
 };
-const setStateHandle = (token) => {
-  sessionStorage.setItem(STATE_HANDLE_SESSION_STORAGE_KEY, token);
+const setStateHandle = (token, appId) => {
+  sessionStorage.setItem(`${STATE_HANDLE_SESSION_STORAGE_KEY}-${appId}`, token);
```

Comment:
add `appId` if only it's defined.. same for remove.

Reply:
It doesn't make a difference if an undefined appId is in the key.

Reply:
difference of `-`

Reply:
yes functionally no difference..